Chart And Axis Titles

jaleskovec opened the pull request:
I've pulled the changes from pull request 137 and added tests and axis titles.

Attached is a sample powerpoint.
[pptx_titles_demo.pptx](https://github.com/scanny/python-pptx/files/46467/pptx_titles_demo.pptx)

jaleskovec commented:
Note that the titles don't appear quite right in LibreOffice vs. PowerPoint. Attached are snapshots.
![libreoffice](https://cloud.githubusercontent.com/assets/15912983/11455215/4d545778-9620-11e5-96bb-20a1427fc0dd.png)
![powerpoint](https://cloud.githubusercontent.com/assets/15912983/11455216/4d5a85f8-9620-11e5-9953-e5c71b4e3737.png)

scanny commented:
Hi @jaleskovec -- this looks quite good actually. Can you segregate it into a separate branch, maybe "feature/cht-title" based on the latest develop branch on scanny/python-pptx and draft up an analysis document especially laying out the API calling protocol, XML examples and relevant schema excerpts? Also if you could describe the "ignored auto... something" bit you mentioned so I can understand what that's about.

From there we should be able to move this along quickly, looks like you have all the tests in place. Nice job :)

jaleskovec commented:
Rebased and added features docs on https://github.com/scanny/python-pptx/pull/198

The "ignore autoTitleDeleted..." commit was pulled in from a different pull request. I think it is referring to a property used to hide the title without removing the title (https://msdn.microsoft.com/en-ca/library/documentformat.openxml.drawing.charts.autotitledeleted(v=office.14).aspx). I don't think we have to worry about it since the title would be set in either case. It looks like it is a property for the tool that displays the powerpoint file.
